src/api/nodeRpcCall.js

```diff
@@ -1242,6 +1251,13 @@ const requestEditCompanyRegistration = async (companyData, companyId, walletAddr
   return submitExtrinsic(extrinsic, walletAddress);
 }
 
+const deleteRequestCompanyRegistration = async (companyId, walletAddress)  => {
```

Comment:
Other PR adds the same thing under different name: https://github.com/liberland/liberland_frontend/pull/162/files#diff-23f1d917744ae222dc753c21f0aea86fd32bbd65ef4e552d1650bba52573b51eR2098-R2103

This seems to be out of scope for this PR though? Why is it here?

Reply:
I fot this is also a part of this PR

Reply:
Okay, will deduplicate this when merging :+1:

Reply:
Other PR is merged, fix please

Reply:
Done.